thirdeye-persistence/src/test/java/ai/startree/thirdeye/datalayer/dao/TestGenericPojoDao.java

```diff
@@ -16,31 +16,48 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import ai.startree.thirdeye.datalayer.MySqlTestDatabase;
+import ai.startree.thirdeye.spi.datalayer.DaoFilter;
 import ai.startree.thirdeye.spi.datalayer.Predicate;
+import ai.startree.thirdeye.spi.datalayer.dto.AnomalyDTO;
 import ai.startree.thirdeye.spi.datalayer.dto.DataSourceDTO;
 import java.util.List;
+import java.util.Random;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 public class TestGenericPojoDao {
 
   private static final String NAME = "name";
+  private static final String BASE_ID = "baseId";
   private static final String TYPE = "type";
   private static final String VERSION = "version";
   private static final List<String> TEST_NAMES = List.of("test1", "test2");
   private static final List<String> TEST_TYPES = List.of("type", "test");
+  private static final int TOTAL_ANOMALIES = 100;
 
   private GenericPojoDao dao;
 
+  private static long getRandomLimit() {
+    return new Random().nextInt(TOTAL_ANOMALIES);
+  }
+
+  private static AnomalyDTO anomaly() {
+    return new AnomalyDTO();
+  }
+
   @BeforeClass
   void beforeClass() {
     dao = MySqlTestDatabase.sharedInjector().getInstance(GenericPojoDao.class);
+    for(int i=0; i<TOTAL_ANOMALIES; i++) {
+      dao.create(anomaly());
+    }
   }
 
   @AfterClass(alwaysRun = true)
   public void afterClass() {
     dao.deleteByPredicate(Predicate.NEQ(NAME, "null"), DataSourceDTO.class);
+    dao.deleteByPredicate(Predicate.NEQ(BASE_ID, 0), AnomalyDTO.class);
```

Comment:
what is this about?

Reply:
Is just cleans up the entries created during the tests